classes/CustomerThread.php

```diff
@@ -151,13 +151,12 @@ public function delete()
                 if (!Validate::isLoadedObject($message)) {
                     $return = false;
                 } else {
-                    $return &= $message->delete();
+                    $return = $return && $message->delete();
                 }
             }
         }
-        $return &= parent::delete();
```

Comment:
Same here, I believe that if `$return` is already false the second part of the logical statement won't be executed in your modification But maybe I'm confusing with shell commands

Reply:
Yep, my bad